pkg/stanza/.github/codecov.yaml

```diff
@@ -0,0 +1,24 @@
+codecov:
```

Comment:
Will this and other .github configuration files be removed in a follow-up PR?

Reply:
Yes, definitely. My next PR will be to do exactly that.

Reply:
I've created #9960 as an aggregate tracking issue.